### ISSUE TITLE
Issue with k8s.io/docs/setup/turnkey/stackpoint/

### ISSUE THREAD

**Issue by alexxa** [kubernetes/website#9479]
**This is a...** 

- [ ] Feature Request
- [x ] Bug Report

**Problem:**
Page [1]  contains too many headers. I believe the page will look better especially its navigation menu, if Headers state Cloud Providers, and  a numerated list is used to describe installation steps, i.e. :

```
## AWS

To create a Kubernetes cluster on AWS, you will need an Access Key ID and a Secret Access Key from AWS.

1. Choose a Provider
    a. Log in to [stackpoint.io](https://stackpoint.io) with a GitHub, Google, or Twitter account.
    b. Click **+ADD A CLUSTER NOW**.
    c. Click to select Amazon Web Services (AWS).

2. Configure Your Provider 
    a. Add your Access Key ID and a Secret Access Key from AWS. Select your default StackPointCloud SSH keypair, or click **ADD SSH KEY** to add a new keypair. 
    b. Click **SUBMIT** to submit the authorization information.

3. Configure Your Cluster
    a. Choose any extra options you may want to include with your cluster
    b. Click **SUBMIT** to create the cluster. 

4. Running the Cluster
   You can monitor the status of your cluster and suspend or delete it from [your stackpoint.io dashboard](https://stackpoint.io/#/clusters).
   For information on using and managing a Kubernetes cluster on AWS, [consult the  Kubernetes documentation](/docs/getting-started-guides/aws/).
```

Also notice that imperative mood is used together with a gerund:
- Choose a Provider
- Configure your Provider
- Configure your Cluster
- Running the Cluster

s/Running/Run will be more consistent.

**Page to Update:**
https://kubernetes.io/docs/setup/turnkey/stackpoint/

**Comment by alexxa** [comment]
@lucperkins I believe it can be closed as fixed by PR #9493

**Comment by Hartmarken348** [comment]
Stckpnt
